### PR TITLE
Toolbar & headline section branch

### PR DESCRIPTION
**Pull request by ClintonYounge**
Features added-

1. The Toolbar and Headline Section; The index.html and style.css code with template styling.
2. Lint config for html and css.
3. A README.md template edited with the necessary information.
4. The background image and additional images.
5. A .gitignore file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+# My Portfolio Project
+A Microverse project to create the Mobile Version of my Portfolio!
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (RaoAkif) [inline README.md]: [OPTIONAL] Your README is perfect 👍🏻 Just remove the Microverse Logo.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Toolbar and Headline Section</title>
```
Comment (RaoAkif) [inline index.html]: [OPTIONAL] Change the title to what the Title of Project is. Just like for example your README. As README won't change with the feature you will be developing.
So You will be assigned custom names only to for example your feature branches, which would only stay relevant until you are done with that feature. But I loved the way you paid attention to even that title 👍🏻

File: style.css [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+  line-height: 20px;
+  align-self: center;
+  color: #6070ff;
+  white-space: nowrap;
+}
+
+header nav div .menu-bar {
+  width: 18px;
+  height: 12px;
+  color: #6070ff;
```
Comment (RaoAkif) [inline style.css]: `#6070ff` color is used 2 times & `#505f79` color is used 3 times in `style.css`. To make the code more DRY, you can create custom colors and use them again and again.
```
  --color-primary: #272a31;
  --color-secondary: #ec5242;
  --color-tertiary: #d3d3d3;
  --color-white: #fff;
  --color-grey: #585858;
  --color-light-grey: #ebf0ee;
```
or like
```
  --primary: #272a31;
  --secondary: #ec5242;
  --tertiary: #d3d3d3;
```

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️

Hi @ClintonYounge,
I really loved your work. But there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Remove Micoverse Logo from README file
- Change the title of images folder from Pics_Needed to some like `images` or `images/icons` etc.
- Try not to repeat the same colors all over the style file. Try to write some DRY code. Create customize variables and use them.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ClintonYounge** [comment]
Thanks a lot for your timely response. I'll get to it!

**Comment by RaoAkif** [comment]
> 

👍🏻

**Comment by ClintonYounge** [comment]
Hello. I made the instructed changes. Please let me know if there's any
more changes needed.

Thanks for your time.

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @ClintonYounge, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**